### PR TITLE
Pass in service names and system services via ENV in development

### PR DESCRIPTION
**Pull request by kevinreedy**
@bellycard/infrastructure @bellycard/platform

### CODE REVIEW

File: lib/configuration.rb [hunk 1]
```diff
@@ -5,9 +5,17 @@ class Configuration
...
+    # no aws?  no problem. Pass in service names and system services via ENV
+    self.service_names = []
+    if ENV['SERVICE_NAMES']
+      self.service_names = ENV['SERVICE_NAMES'].split(',')
+    end
+
     self.system_services = []
+    if ENV['SYSTEM_SERVICES']
+      self.system_services = ENV['SYSTEM_SERVICES'].split(',')
+    end
```
Comment (shaqq) [inline lib/configuration.rb]: these are hard-coded into Consul?

Comment (kevinreedy) [inline lib/configuration.rb]: In production, they are passed in via [ec2 user data](https://github.com/bellycard/docker_consul_update/blob/master/lib/configuration.rb#L13-L21). Previously in dev they were hard-coded in code. I'm changing it to be based in by ENV variables.

Comment (shaqq) [inline lib/configuration.rb]: gotcha

**Comment by shaqq** [comment]
+1 from yo

**Pull request merged**